src/pages/lock-exception/LockException.js

```diff
@@ -520,18 +524,16 @@ class LockException extends Page {
                                 pageState: newPageState,
                             })
                         } else {
-                            this.props.deleteInProgress = true
-                            this.context.updateAppState({
-                                pageState: newPageState,
-                            })
+                            this.deleteInProgress = true
                             this.loadLockExceptionsForPager(
                                 LockException.initialPager,
-                                false
+                                true
```

Comment:
Passing `true` as second arg to `loadLockExceptionsForPager` ensures a fresh list is fetched and this helper will also ensures the page state is correct.